source/core/CEntryPoints.cpp

```diff
@@ -184,6 +184,18 @@ VIREO_EXPORT EggShellResult EggShell_FindSubValue(TypeManagerRef tm,
     return kEggShellResult_Success;
 }
 //------------------------------------------------------------
+//! Check if a value has been written by Vireo (and reset the state)
+VIREO_EXPORT Boolean EggShell_TestAndResetNeedsUpdateFlag(TypeManagerRef tm, const TypeRef typeRef, const void* pData)
```

Comment:
Please change the name of the function to something like:
EggShell_IsUpdateNeededAndReset and the next one to
EggShell_IsUpdateNeeded

Reply:
We already had a discussion about this, and landed on this name because 'TestAndReset' is a well-defined term of art describing this behavior.  Your proposed name is awkward, combining a question predicate with a verb.
Do you have a suggestion for an improved name that retains the 'TestAndReset' phrase?

Reply:
Ok, let's keep this name.